src/components/chipsInput/index.tsx

```diff
@@ -146,6 +147,37 @@ const ChipsInput = forwardRef((props: ChipsInputProps, refToForward: React.Ref<a
     );
   }, [chips, leadingAccessory, defaultChipProps, removeMarkedChip, markedForRemoval, maxChips]);
 
+  const isRequired = useMemo(() => {
+    if (!validate) {
+      return false;
+    }
+
+    const inputValidators = Array.isArray(validate) ? validate : [validate];
+    return inputValidators.includes('required');
+  }, [validate]);
+
+  const requiredValidator = useCallback(() => {
+    return !isRequired || (chips?.length ?? 0) > 0;
+  }, [isRequired, chips]);
+
+  const _validate = useMemo(() => {
+    if (!validate) {
+      return undefined;
+    } else if (isRequired) {
+      const inputValidators = Array.isArray(validate) ? validate : [validate];
+      return inputValidators.map(validator => (validator === 'required' ? requiredValidator : validator));
+    } else {
+      return validate;
+    }
+  }, [validate, isRequired, requiredValidator]);
+
+  useDidUpdate(() => {
+    if (others.validateOnChange) {
```

Comment:
What about 'validateOnStart' and 'validateOnBlur' props? Are we not supporting them here?

Reply:
They're working as far as I've tested